Configuration/PyReleaseValidation/python/MatrixInjector.py

```diff
@@ -647,6 +653,6 @@ def submit(self):
                 workFlow=makeRequest(self.wmagent,d,encodeDict=True)
                 print("...........",n,"submitted")
                 random_sleep()
-            
-
-        
+        if self.testMode and self.countLongWFName>0:
+            print("\n*** FIX NEEDED BEFORE INJECTION: "+str(self.countLongWFName)+" candidate workflows have too long name (>81 characters) ***")
+            print(self.longWFName)
```

Comment:
`print('\n'.join(self.longWFName))` (once converted to a list)

Reply:
Done. Thanks.